contexts/arbo-context.tsx

```diff
@@ -61,44 +62,16 @@ function filterData(data: any[], filters: { [key: string]: string[] }): any[] {
   });
 }
 
-export function setMapboxFilters(
-  filters: { [key: string]: string[] },
-  map: mapboxgl.Map | null,
-) {
-  let mapboxFilters: any = [];
-
-  Object.keys(filters).forEach((filter: string) => {
-    const keyFilters: any = [];
-    if (filters[filter].length > 0) {
-      if(filter === "antibody") {
-        filters["antibody"].forEach((antibody: string) => {
-          keyFilters.push([">", ["index-of", antibody, ["get", "antibody"]], -1]);
-        });
-      } else {
-        filters[filter].forEach((filterValue: string) => {
-          keyFilters.push(["in", filterValue, ["get", filter]]);
-        });
-      }
-    }
-    if (keyFilters.length > 0) mapboxFilters.push(["any", ...keyFilters]);
-  });
-
-  console.log("Map Filters: ", ["all", ...mapboxFilters]);
-  if(map?.getLayer("Arbovirus-pins")) map?.setFilter("Arbovirus-pins", ["all", ...mapboxFilters]);
-}
-
-export const arboReducer = (state: ArboStateType, action: ArboAction) => {
+export const arboReducer = (state: ArboStateType, action: ArboAction, map: MapRef | undefined) => {
   switch (action.type) {
     case ArboActionType.UPDATE_FILTER:
       const selectedFilters = {
         ...state.selectedFilters,
         [action.payload.filter]: action.payload.value,
       };
 
-      if (action.payload.map) {
-        setMapboxFilters(selectedFilters, action.payload.map);
```

Comment:
I'm piping the filtered data down so doing anything additional with the mapbox filters is unecessary.